scripts/release.ts

```diff
@@ -224,6 +224,9 @@ async function main() {
     }
 
     fs.writeFileSync(packageJsonPath, pkg);
+    if (packageName === 'tfjs-node-gpu') {
+      $(`yarn prep-gpu`);
```

Comment:
move this to "scripts" of the Node phase

if this runs for both packages, let's make the scripts field a map

Reply:
Update the scripts become an object of:
```
{
  package-name: {
    before-yarn: ['script'],
    after-yarn: ['script']
  }
}